.github/actions/npm-install-test/action.yml

```
@@ -43,12 +47,36 @@ runs:
         # deprecated
         npm run downloadTypes --if-present
 
-        if [[ $(jq '.scripts["test:coverage:ci"]' < package.json;) != null && "${{ inputs.withoutCoverage }}" != 'true' ]]; then
-          npm run test:coverage:ci
-          echo "INCLUDE_COVERAGE_REPORT=true" >> $GITHUB_OUTPUT
-        elif [[ $(jq '.scripts["test:coverage"]' < package.json;) != null && "${{ inputs.withoutCoverage }}" != 'true' ]]; then
-          npm run test:coverage
-          echo "INCLUDE_COVERAGE_REPORT=true" >> $GITHUB_OUTPUT
-        else
-          npm run test --if-present
+        maxRuns=$(( ${{ inputs.retries }} + 1 ))
+        runCounter=1
+
+        until [ $runCounter -gt $maxRuns ]
+        do
+          if [ $runCounter -gt 1 ]; then
```

```suggestion
        until [[ $runCounter -gt $maxRuns ]]
        do
          if [[ $runCounter -gt 1 ]]; then
```

Let's use `[[  ]]` instead of `[ ]`  we're already using it in the rest of the PR and the former is more advanced and less error prone than the latter.